dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports-reconciliation-summary.controller/helpers.ts

```diff
@@ -159,9 +159,10 @@ export const generateReconciliationSummaries = async (currentSubmissionMonth: Is
   // TODO FN-1949 - when banks' reporting schedules are added to the `bank` collection, add new repo method to only
   //  fetch banks that are due to submit this month
   const banks = await getAllBanks();
+  const banksVisibleInTfm = banks.filter((bank) => bank.isUtilisationReportVisibleInTfm);
```

Comment:
If `bank.isUtilisationReportVisibleInTfm` is not defined does it just default to false?

Reply:
It is expected to always be defined...*but*, if it isn't defined, this will treat it the same as `false`